otherlibs/unix/envir.c

```diff
@@ -13,18 +13,51 @@
 /*                                                                        */
 /**************************************************************************/
 
+#include "caml/config.h"
```

Comment:
Good point.  The convention seems to be to use `"caml/config.h"` in the distribution proper, and `<caml/config.h>` elsewhere (e.g. `otherlibs`).  I'll update this.